cypress/utils/utils.ts

```diff
@@ -1859,6 +1859,7 @@ export function validateTackleCr(): void {
 }
 
 export function validateTackleOperatorLog(): void {
+    cy.wait(30 * SEC);
```

Comment:
You shouldn't just add wait like this. Let's try increasing timeout in another place:
```
export function getCommandOutput(command: string): Cypress.Chainable<Cypress.Exec> {
    return cy.exec(command, { timeout: 30 * SEC }).then((result) => {
        return result;
    });
}
```
You can try to increase timeout in `cy.exec()` so that it will wait longer for the command to work.

Reply:
Thanks @ibragins.
The thing is that the command is working but there is no output, it didn't fail on timeout.

Reply:
Got you, that is something different... Does it work for you well when adding timeout like this?

Reply:
Yes, here is the Jenkins run:
https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-ui-tests-runner/2999/